docs/ReleaseNotes/ReleaseNotes.Atom.May.2019.md

```diff
@@ -0,0 +1,32 @@
+# Atom Release (May 2019)
+
+!!! warning
+	- Due to significant data format changes, you must perform a full install of the Atom version. You cannot perform an upgrade process and retain your existing data. 
+	- To copy over your data to a CM-Well node with the new version, either re-ingest your data, or perform a DC-Sync process between the "old node" and the "new node". 
+	- After installing Atom, you will be able to perform an upgrade process for subsequent releases.
+
+Title | Git Issue | Description 
+:------|:----------|:------------
+Major version upgrades to 3rd-party libraries | [1003](https://github.com/thomsonreuters/CM-Well/pull/1003), [1109](https://github.com/thomsonreuters/CM-Well/pull/1109) | Several 3rd-party libraries have been upgraded, including: Elasticsearch -> 7, Cassandra -> 3.11.4, Kafka -> 2.1.1. **NOTE that due to the significant data format changes caused by these upgrades, you must perform a full reinstall of CM-Well; you CANNOT perform an upgrade process that retains the data.**
```

Comment:
do we need that note here, given the one above?

Reply:
I shortened this note.